docs/source/devguide/contributing.rst

```diff
@@ -0,0 +1,114 @@
+new members to the community and will do our best to help you get up to speed.
+The purpose of this section is to document how the project is managed: how
+contributions (bug fixes, enhancements, new features) are made, how they are
+evaluated, who is permitted to merge pull requests, and what happens in the
+event of disagreements. Once you have read through this section, the
+:ref:`devguide_workflow` section outlines the actual mechanics of making a
+contribution (forking, submitting a pull request, etc.).
+
+The goal of our governance model is to:
+
+- Encourage new contributions.
+- Encourage contributors to remain involved.
+- Avoid unnecessary processes and bureaucracy whenever possible.
+- Create a transparent decision making process which makes it clear how
+  contributors can be involved in decision making.
+
+Overview
+--------
+
+OpenMC uses a liberal contribution model for project governance. Anyone involved
+in development in a non-trivial capacity is given an opportunity to influence
+the direction of the project. Project decisions are made through a
+consensus-seeking process rather than by voting.
+
+Terminology
+-----------
+
+- A *Contributor* is any individual creating or commenting on an issue or pull
+  request.
+- A *Committer* is a subset of contributors who are authorized to review and
+  merge pull requests.
+- The *TC* (Technical Committee) is a group of committers who have the authority
+  to make decisions on behalf of the project team in order to resolve disputes.
+- The *Project Lead* is a single individual who has the authority to make a final
+  decision when the TC is unable to reach consensus.
+
+Contribution Process
+--------------------
+
+Any change to the OpenMC repository must be made through a pull request (PR).
+This applies to all changes to documentation, code, binary files, etc. Even long
+term committers and TC members must use pull requests.
+
+No pull request may be merged without being reviewed.
+
+For non-trivial contributions, pull requests should not be merged for at least
+36 hours to ensure that contributors in other timezones have time to review.
+Consideration should also be given to weekends and other holiday periods to
+ensure active committers have reasonable time to become involved in the
+discussion and review process if they wish.
+
+The default for each contribution is that it is accepted once no committer has
+an objection. During review committers may also request that a specific
+contributor who is most versed in a particular area review the PR before it can
+be merged. Once all issues brought by committers are addressed it can be merged
+by any committer.
+
+In the case of an objection being raised in a pull request by another committer,
+all involved committers should seek to arrive at a consensus by way of
+addressing concerns being expressed through discussion, compromise on the
+proposed change, or withdrawal of the proposed change.
```

Comment:
I think this paragraph and the one above can be shortened and merged together with something like:

> A pull request can be merged by any committer, but only if no objections are raised by any other committer. In the case of an objection being raised, all involved committers should seek consensus through discussion and compromise.

That would remove the first sentence of the first paragraph which I think can be easily misunderstood (someone might think a merge is automatic at 36 hours if no review is made).  I think the bit about requesting a specific contributor (second sentence, first paragraph) will probably be understood without needing to be explicitly written down.